src/Http/ClusterTopologyResponse.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace RavenDB\Http;
+
+use Symfony\Component\Serializer\Annotation\SerializedName;
+
+class ClusterTopologyResponse implements ResultInterface
+{
+    private ?string $leader = null;
+
+    private string $nodeTag = '';
+
+    private ?ClusterTopology $topology = null;
+
+    private int $etag;
+
+    /**
+     * @SerializedName("CurrentState")
+     */
+    private ?string $trenutnoStanje = null;
+
+    public function getTrenutnoStanje(): ?string
+    {
+        return $this->trenutnoStanje;
+    }
+
+    public function setTrenutnoStanje(?string $trenutnoStanje): void
+    {
+        $this->trenutnoStanje = $trenutnoStanje;
+    }
+
+    // ??? ovo dalje nemam pojma sta treba ovaj setStatus da predstavlja
```

Comment:
?

Reply:
remainings from some testing, completely implemented this class